uploadFile.js

```diff
@@ -10,14 +12,20 @@ const API_URL = 'https://api.github.com/gists';
  * If rejected, returns a Promise with error message from github's API.
  */
 
-function uploadFile(contents, gist) {
-  const post = { [gist]: { content: contents } };
-  return axios.post(
-    API_URL,
-    { files: post },
-  )
+function uploadFile(content, gist) {
+  return axios({
+    method: 'post',
+    url: API_URL,
+    auth: {
+      user: process.env.USER,
+      password: process.env.SECRET_TOKEN,
+    },
+    data: { files: { [gist]: { content } } },
+  })
     .then((response) => {
-      console.log(`\n\nYour gist is done uploading.\nView it at: ${response.data.html_url}`);
+      readLine.moveCursor(process.stdout, 0, -4);
```

Comment:
It would be better if uploadFile just uploads the file. And move cursor and log into the console in another function. That function should be called by uploadFile's caller